<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IstenenTetkikEkle</class>
 <widget class="QDialog" name="IstenenTetkikEkle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>307</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>321</width>
    <height>307</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tetkik İste</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QComboBox" name="cbTetkikdurumu">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>İstendi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Örnek Alındı</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Çalışılıyor</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sonuçlandı</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>İptal Edildi</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="2" column="0">
    <widget class="QDateTimeEdit" name="dteIstekTarihi">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btnKaydet">
     <property name="text">
      <string>Tetkik İste</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QComboBox" name="cbZiyaretler"/>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="btnVazgec">
     <property name="text">
      <string>Vazgeç</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbZiyaretler</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>dteIstekTarihi</tabstop>
  <tabstop>cbTetkikdurumu</tabstop>
  <tabstop>btnKaydet</tabstop>
  <tabstop>btnVazgec</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnVazgec</sender>
   <signal>clicked()</signal>
   <receiver>IstenenTetkikEkle</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
